src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -38,7 +38,7 @@ public class TokenValidationParameters
         /// Default for the maximum token size.
         /// </summary>
         /// <remarks>250 KB (kilobytes).</remarks>
-        public const Int32 DefaultMaximumTokenSizeInBytes = 1024 * 250;
+        public const int DefaultMaximumTokenSizeInBytes = 1024 * 250;
```

Comment:
Is it a breaking change?

Reply:
@jmprieur Good question.
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/built-in-types

I believe they are the same, similar to String and string.